docker/test

```diff
@@ -11,7 +11,7 @@ USAGE: test [OPTIONS]
 OPTIONS:
   -c CMD    sets the command to be run by the container
               default: $TEST_CMD
-  -m        use mysql
+  -m [OS]   use mysql, we can specify OS where candlepin will be running. Default is Centos Stream 8: cs8.
```

Comment:
If we need to specify the OS/image version, It would be much better to use a completely new argument (e.g. `-a [image]` instead of making this an argument on the mysql flag

Reply:
I know we talked about it, but then I thought about it and I thought that there are a lot of edge cases that we'll have to cover(both parameters need to be set, different OS for mariadb and postgres and more ) and I don't know who all uses the script, so this seemed like a safe choice, but I can redo it if you want.

Reply:
I think we should re-do it by using a separate flag. This is not intuitive and will confuse people, because one flag is used for specifying two arguments (db, and image)

Reply:
done